atomic_cms.gemspec

```diff
@@ -7,6 +7,7 @@ Gem::Specification.new do |s|
   s.authors     = ['Don Humphreys']
```

Comment:
Awesome! As far as I can tell, the only published gem is atomic_assets. I think we should register a gem for the atomic_cms in order to hold the name while we're waiting to publish this.